### PR TITLE
Matrixsorting issue825

### PR DESCRIPTION
**Pull request by narayan2111**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Matrix sorting aims to sort the elements of the given matrix without affecting the matrix structure.
The algorithm takes as input a 2D array of elements and sorts it in the desired order. The elements are present in the desired order once the algorithm stops.

### Checklist

- [Y ] I've read the contribution guidelines.
- [ Y] I've checked the issue list before deciding what to submit.
- [ Y] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#825 
ProgramOutput

### CODE REVIEW

**Comment by narayan2111** [comment]
Kindly review this PR @SKAUL05 @iamrajiv @plazzy99 @atarax665

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]
```
Input:
3 2
7 8 9 5 6 4

Sorted Matrix: 
5 7 
6 4 
6 4 
````
test case failed

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@narayan2111 remove unnecessary line breaks and indentations

**Comment by narayan2111** [comment]
I Updated the required changes @plazzy99 

Output

**Comment by narayan2111** [comment]
> @narayan2111 remove unnecessary line breaks and indentations

Done Sir

File: C/ds/MatrixSorting.c [hunk 1]
```diff
@@ -0,0 +1,87 @@
+/*C implementation to sort the given matrix
+
+ Input :
+  Enter Row: 3
+  Enter Column: 3
+  Enter the elements of matrix :
+  7 8 9
+  5 6 4
+  3 1 2
```
Comment (vatsalkesarwani12) [inline C/ds/MatrixSorting.c]: place this comment at the rest is fine

Comment (narayan2111) [reply C/ds/MatrixSorting.c:9]: where to place this comment

Comment (narayan2111) [reply C/ds/MatrixSorting.c]: Done Sir.

**Review by vatsalkesarwani12 — APPROVED** [review]
LGTM 👍

**Review by iamrajiv — APPROVED** [review]

File: C/README.md [hunk 2]
```diff
@@ -17,9 +17,7 @@
 - [Queue Implementation with Linked List](ds/QueueImplementationWithList.c)
 - [Singly Linked List Implementation](ds/singly_linked_list.c)
 - [Interpolation Search](search/Interpolation.c)
-- [Spiral Matrix](ds/Spiral_Matrix.c)
```
Comment (atarax665) [inline C/README.md:38]: No need of deleting existing hyperlinks.

File: C/README.md [hunk 3]
```diff
@@ -17,9 +17,7 @@
...
-
-
+- [Matrix Sorting ](ds/MatrixSorting.c)
```
Comment (atarax665) [inline C/README.md]: There is no MatrixSorting.c file in ds folder. Remove this line.

File: C/README.md [hunk 4]
```diff
@@ -35,6 +33,8 @@
 - [Merge Sort](sort/MergeSort.c)
 - [OddEven Sort](sort/oddevensort.c)
 - [Selection Sort](sort/selection_sort.c)
+- [Matrix Sort](sort/MatrixSorting.c)
```
Comment (atarax665) [inline C/README.md:39]: You have added the link of same program twice, remove this line.

File: C/README.md [hunk 5]
```diff
@@ -70,76 +70,3 @@ _add list here_
 - [Number_of_duplicates_using_Binary_search](other/Number_of_duplicates_using_Binary_search.c)
 _add list here_
```
Comment (atarax665) [inline C/README.md]: Remove this line too.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: C/README.md (hunk 5 above)
Comment (narayan2111) [reply C/README.md]: Done Sir @atarax665

Comment (narayan2111) [reply C/README.md]: kindly review this PR @atarax665

**Review by atarax665 — APPROVED** [review]
I have fixed the README.md file.

**Pull request merged**